src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -566,6 +586,13 @@ limitations under the License.
       <arg name="value" type="int64u"/>
     </command>
 
+    <command source="server" code="0x0C" name="TestBatchHelperResponse" optional="true" disableDefaultResponse="true" apiMaturity="provisional">
+      <description>
+        Delivers the ID of an event in response to test command emitting fabric scoped events.
+      </description>
+      <arg name="buffer" type="octet_string" length="800"/>
```

Comment:
Does the command work if the max is not used?

Reply:
Yes it does. I think that length is supposed to indicate max. Although it isn't clear to me where enforcement would take place, as I don't see anything on the encode.

I used `CertificateChainResponse` in operations credential cluster as a reference src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml